content/rs/references/cli-utilities/crdb-cli/crdb/add-instance.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: crdb-cli crdb add-instance
+linkTitle: add-instance
+description: Adds a peer replica to an Active-Active database.
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases:
+---
+
+Adds a peer replica to an existing Active-Active database in order to host the database on another cluster. This creates a read-write copy of the database on the specified cluster.
```

Comment:
how about saying "... This creates an additional active instance of the database..."